docs/how_to_guide/categorical_predictors.md

```diff
@@ -0,0 +1,120 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Construct Design Matrices for Categorical Features
+
+## Design Matrix with `patsy`
+
+If you have categorized your trials and want to capture a change in the firing rate with the trial category, you can create your design matrix using [`patsy`](https://patsy.readthedocs.io/en/latest/).
+
+Let's assume we have a simple dataset with four samples and two labels assigned to each sample. You can think of these labels as characteristics of the trial to which the samples belong, such as stimulus 1 vs stimulus 2 or context 1 vs context 2.
+
+```{code-cell} ipython3
+# Example data
+import pandas as pd
+from patsy import dmatrix
+import numpy as np
+
+# Example data 
+data = pd.DataFrame({
```

Comment:
this example is very abstract. I think we'd better off using specific stimulus / context, and pointing out that the general principle applies

Reply:
I tried to make the stim and context more concrete, let me know if it works

Reply:
I think this is good!